### DISCUSSION TITLE
Utilities for parsing Link headers

### DISCUSSION THREAD

**Discussion by Midnighter** [encode/httpx#2687] [General] [answered]
This was originally raised in https://github.com/encode/httpx/issues/205 and I'm wondering if anything in that regard was ever implemented? If not as an attribute on the response object, maybe some utility functions? I couldn't find anything in the docs or other issues/discussions.

**Comment by lovelydinosaur** [comment, marked as answer]
>  I'm wondering if anything in that regard was ever implemented?

no there wasn't ever a `.links` attribute or similar.
